io.catenax.fleet.diagnostic_data/2.0.0/DiagnosticData.ttl

```diff
@@ -0,0 +1,541 @@
+
+:recordStatus a samm:Property ;
+   samm:preferredName "Record status"@en ;
+   samm:description "The record operation enumeration can be used to realize delta update concept.\nDelta update concept means:\n- You transfer an initial load of data\n- After the first week only the delta to the initial load is transferred\n\n\nThe record status describes whether this record is:\n- new=>This record is transferred the first time\n- update=> Some properties of this record have changed compared to a previous transfer\n- delete=> This record was transferred in the initial load or in a previous delta update, but is not \n used any more and therefore it should be deleted on data consumer side\n- same=> This record was transferred in the initial load or in a previous delta update"@en ;
+   samm:characteristic :RecordStatusEnumeration ;
+   samm:exampleValue "update" .
+
+:sessionId a samm:Property ;
+   samm:preferredName "Session identifier"@en ;
+   samm:description "A oem-specific session identifier that is unique in this company."@en ;
+   samm:characteristic :UniqueID ;
+   samm:exampleValue "3747429FGH382923974682_Session_2023-11-04T08:00:45" .
+
+:anonymizedVIN a samm:Property ;
+   samm:preferredName "Anonymized VIN"@en ;
+   samm:description "OEM-specific hashed VIN. Unique within OEM."@en ;
+   samm:characteristic :UniqueID ;
+   samm:exampleValue "3747429FGH382923934abcf74682" .
+
+:catenaXQualityTaskId a samm:Property ;
+   samm:preferredName "Quality Task ID"@en ;
+   samm:description "The fully anonymous and unique Catena-X ID of this quality task. This id is unique inside Catena-X data space."@en ;
+   samm:characteristic ext-uuid:UuidV4Trait ;
+   samm:exampleValue "430f56d3-1234-1234-1234-aaaabbbbcccc" .
+
+:catenaXVehicleId a samm:Property ;
+   samm:preferredName "Vehicle Catena-X Identifier"@en ;
+   samm:description "A unique identifier for the vehicle in the Catena-X data space."@en ;
+   samm:characteristic ext-uuid:UuidV4Trait ;
+   samm:exampleValue "580d3adf-1978-44a0-a214-13d6ceed9379" .
+
+:countryCode a samm:Property ;
+   samm:preferredName "Country code"@en ;
+   samm:description "Country code in  ISO 3166-1 alpha-3 codes where this session took place"@en ;
+   samm:characteristic :CountryCodeTrait ;
+   samm:exampleValue "DEU" .
+
+:diagnosticSoftwareName a samm:Property ;
+   samm:preferredName "Diagnostic Software"@en ;
+   samm:description "The name of the diagnostic software that was used."@en ;
+   samm:characteristic samm-c:Text ;
+   samm:exampleValue "Pontiac Car Diagnostic software" .
+
+:diagnosticSoftwareVersion a samm:Property ;
+   samm:preferredName "Diagnostic software version"@en ;
+   samm:description "The version of the used diagnostic software"@en ;
+   samm:characteristic samm-c:Text ;
+   samm:exampleValue "V.411 Patch 2023-02" .
+
+:mileage a samm:Property ;
```

Comment:
Property and the referenced Characteristic should not have the same name

Reply:
fixed with next upload